src/modules/job-list/list.c

```diff
@@ -68,7 +70,9 @@ int get_jobs_from_list (json_t *jobs,
         if (job->t_inactive > 0. && job->t_inactive <= since)
             break;
 
-        if (job_match (job, c)) {
+        if ((ret = job_match (job, c, &error)) < 0)
+            return -1;
```

Comment:
This function uses a local `flux_error_t` to handle the error message from `job_match()`, but doesn't copy it to `errp->text` on error. Should `errp` be used here and the local `flux_error_t error` be dropped?

Reply:
Oh, this also indicates that there's no testing of a proper error message in the response when `job_match()` fails.

Reply:
good catch, I have tests to ensure the error occurs but didn't check the message